overrides/default.nix

```diff
@@ -1946,6 +1946,10 @@ lib.composeManyExtensions [
         }
       );
 
+      pynetbox = super.pynetbox.overridePythonAttrs (old: {
```

Comment:
Why isn't this added in the build systems json file?

Reply:
@KiruyaMomochi Any thoughts here?

Reply:
Sorry I think I have sent the comment but it have stayed in pending state for one week...

Reply:
Ah, I guess I missed that!